gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/drilling/DrillDown.java

```diff
@@ -56,15 +53,15 @@ public DrillDown(
   public LogicalGraph execute(LogicalGraph graph) {
     if (getElement() == Element.VERTICES) {
       graph = graph.transformVertices(
-        new DrillDownTransformation<Vertex>(getLabel(), getPropertyKey(), getVertexDrillFunction(),
+        new DrillDownTransformation<>(getLabel(), getPropertyKey(), getVertexDrillFunction(),
```

Comment:
for readability i prefer a switch case here 
```
    switch (getElement()) {
    case VERTICES:
      graph = graph.transformVertices(
        new DrillDownTransformation<>(
          getLabel(),
          getPropertyKey(),
          getVertexDrillFunction(),
          getNewPropertyKey(),
          drillAllLabels(),
          keepCurrentPropertyKey()));
      break;
    case EDGES:
      graph = graph.transformEdges(
        new DrillDownTransformation<>(
          getLabel(),
          getPropertyKey(),
          getEdgeDrillFunction(),
          getNewPropertyKey(),
          drillAllLabels(),
          keepCurrentPropertyKey()));
      break;
    case GRAPHHEAD:
      graph = graph.transformGraphHead(
        new DrillDownTransformation<>(
          getLabel(),
          getPropertyKey(),
          getGraphheadDrillFunction(),
          getNewPropertyKey(),
          drillAllLabels(),
          keepCurrentPropertyKey()));
      break;
      default: throw new UnsupportedTypeException("Element type must be vertex, edge or graphhead");
```

Reply:
Done.